Description: Fix missing .py files
 It's looking like upstream MANIFEST.in is wrong and doesn't package some of
 the .py files inside oslo_concurrency.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-12-23

--- python-oslo.concurrency-0.3.0.orig/MANIFEST.in
+++ python-oslo.concurrency-0.3.0/MANIFEST.in
@@ -2,5 +2,6 @@ include AUTHORS
 include ChangeLog
 exclude .gitignore
 exclude .gitreview
+recursive-include oslo_concurrency *.py
 
 global-exclude *.pyc
\ No newline at end of file
